Code/Editor/Core/LevelEditorMenuHandler.cpp

```diff
@@ -758,24 +758,24 @@ QMenu* LevelEditorMenuHandler::CreateHelpMenu()
     layout->addWidget(lineEdit);
     containerWidget->setLayout(layout);
     containerWidget->setContentsMargins(2, 0, 2, 0);
-    lineEdit->setPlaceholderText(tr("Search documentation"));
+    lineEdit->setPlaceholderText(tr("Search documentation..."));
     lineEditSearchAction->setDefaultWidget(containerWidget);
 
     auto searchAction = [lineEdit]()
         {
             auto text = lineEdit->text();
             if (text.isEmpty())
             {
-                QDesktopServices::openUrl(QUrl("https://o3de.org/docs/"));
+                QDesktopServices::openUrl(QUrl("https://www.o3de.org/docs/"));
```

Comment:
is the `www` necessary?

Reply:
It is not necessary, but it is the correct url (trying to access `https://o3de.org/` will redirect to `https://www.o3de.org/`). The real fix here is on the line below, switching from `/docs/` to /`search/`.